### ISSUE TITLE
errors.com.epicgames.accountportal.captcha_invalid

### ISSUE THREAD

**Issue by jokoast** [SzymonLisowiec/node-epicgames-client#99]
The example sample with my own credentials returns 
`errors.com.epicgames.accountportal.captcha_invalid`

Don't know how to bypass that.

**Comment by davispuh** [comment]
Yeah it always fails with this, even if you login manually and solve captcha it doesn't remember that for next time.

**Comment by PaulKovalov** [comment]
any solution for this?

**Comment by davispuh** [comment]
Kinda, save your session after have logged in so you don't need to relogin and do captcha again. Also this repo seems abandoned, we're maintaining a fork here https://github.com/Revadike/node-epicgames-client

**Comment by ThisNils** [comment]
Logging in via email and password is not recommended anymore as you'll get a captcha 90% of the time. Unfortunately this library does not support device auths. If you only need Fortnite functionality, you could take a look at my library https://fnbr.js.org/#/ which is actively maintained and supports auth methods like device auths and launcher refresh tokens
